csi/node_server.go

```diff
@@ -812,11 +812,51 @@ func getNodeServiceCapabilities(cs []csi.NodeServiceCapability_RPC_Type) []*csi.
 	return nscs
 }
 
+func (ns *NodeServer) requireHostNamespaceMounter(volume *longhornclient.Volume, volumeCapability *csi.VolumeCapability) (bool, error) {
+	if !requiresSharedAccess(volume, volumeCapability) {
+		return false, nil
```

Comment:
return true?

Reply:
I am intending to return `false` because I only want to enter host name space for regular RWX when RWX storage network is not enable.

For RWO volume, I think we don't need to enter host namespace

This is similar to the logic previously https://github.com/longhorn/longhorn-manager/blob/af348f2432967cd2981582cddb8129cab3826919/csi/node_server.go#L823-L827 

WDYT?

Reply:
I see. Thank you.